.github/workflows/pytest-gpu.yml

```diff
@@ -34,12 +34,15 @@ jobs:
           arch: "clang"
           platform: "nvidiaX"
           language: "openmp"
+          # FIXME: Re-add and "gpu" and amend tags before merge.
```

Comment:
need to address this?

Reply:
It won't cause problems but I'll need to mess around with a couple of runners after the merge. This comment can now be removed.